python/pipeline/utils/h5.py

```diff
@@ -94,6 +94,106 @@ def read_behavior_file(filename):
 
     return data
 
+def read_patchmaster(F):
+    if type(F)==str:
+        F=F[:-4]+'%d.h5' # chanage oneCell10_01.h5 to oneCell10_%d.h5
+        # Append '%d.h5' to filename in place of trailing 'x.h5'
+        # right now this way will only work for one digit file names, are we doing two digit names in the future? if so i suggest use the 0d format instead of having 9 and suddently 10 for eg.
+    else:
+        raise Exception ('currently the input format as {} is not surrpoted, please use str'.format(type(F)))
+    f=h5py.File(F,"r+",driver='family',memb_size=0) # read only, mimic if we are reading a batch of files, but the batch only have 0 member size
+    #begin case 7
+    '''
```

Comment:
if this is documentation you can put it under read_patchmaster(F):
```python
def read_patchmaster(F):
    """ Read patch clamp data recorded with ....

    < More info about the function (if any) >

     :param string F: Filename .......

     :returns: ......
    """
```
check `read_behavior_file` in this same file for an example